apps/openmw/mwworld/store.cpp

```diff
@@ -294,6 +294,18 @@ namespace MWWorld
         return true;
     }
 
+    template<>
+    bool Store<ESM::Dialogue>::eraseStatic(const std::string &id)
+    {
+        auto it = mStatic.find(Misc::StringUtils::lowerCase(id));
+
+        if (it != mStatic.end() && Misc::StringUtils::ciEqual(it->second.mId, id)) {
+            mStatic.erase(it);
+        }
+
+        return true;
```

Comment:
I suppose it would be better to move this override to other dialogue overrides (near line 1050).

Reply:
Maybe. I think there's something to be said for keeping them together so you can quickly see it's being overridden though.

Reply:
Move it! Some Store things are already being overridden for dialogue, no need to be inconsistent.